package.json

```diff
@@ -38,7 +37,17 @@
     "husky": "^4.2.5",
     "lint-staged": "^10.2.13",
     "prettier": "^2.1.1",
-    "pretty-quick": "^3.0.0"
+    "pretty-quick": "^3.0.0",
+    "react-test-renderer": "^17.0.1"
```

Comment:
I guess react-test-renderer works fine for snapshots, but I would suggest you try to use only react testing library instead. Snapshot testing can be the same as having no tests. (: